### PR TITLE
Update aircrack-ng.rb 

### PR DESCRIPTION
**Pull request by souvik1997**
Fixes #29450 

Follow-up from earlier PR at #42762.

### CODE REVIEW

**Comment by mikemcquaid** [comment]
Squash commits by running `brew update; git checkout $YOUR_BRANCH; git rebase --interactive origin/master` and then mark each commit other than the first as `squash` or `fixup` then `git push -f`. Thanks!

**Comment by souvik1997** [comment]
Well it looks like upstream has fixed the issue: I've updated the formula for 1.2rc3: 4d743a3

File: Library/Formula/aircrack-ng.rb [hunk 1]
```diff
@@ -1,26 +1,21 @@
 class AircrackNg < Formula
   desc "Next-generation aircrack with lots of new features"
   homepage "http://aircrack-ng.org/"
-  # We can't update this due to linux-only dependencies in >1.1.
-  # See https://github.com/Homebrew/homebrew/issues/29450
-  url "http://download.aircrack-ng.org/aircrack-ng-1.1.tar.gz"
-  sha256 "b136b549b7d2a2751c21793100075ea43b28de9af4c1969508bb95bcc92224ad"
+  url "http://download.aircrack-ng.org/aircrack-ng-1.2-rc2.tar.gz"
```
Comment (mikemcquaid) [inline Library/Formula/aircrack-ng.rb]: This needs to be a stable, non-beta/RC version.

File: Library/Formula/aircrack-ng.rb [hunk 2]
```diff
@@ -1,26 +1,21 @@
...
+  sha256 "ba5b3eda44254efc5b7c9f776eb756f7cc323ad5d0813c101e92edb483d157e9"
   revision 1
```
Comment (mikemcquaid) [inline Library/Formula/aircrack-ng.rb]: Please remove this `revision` line on version updates.

**Comment by mikemcquaid** [comment]
This PR needs to be rebased on `master`

**Comment by apjanke** [comment]
Needs a rebase and probably a version bump from upstream. Closing out. If this still needs to be done, please re-submit on the new homebrew/core repo.

**Pull request closed without merging**